tests/Integration/Behaviour/Features/Scenario/TaxRulesGroup/tax_rule_group_management.feature

```diff
@@ -0,0 +1,56 @@
+@restore-all-tables-before-feature
+#./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s tax_rules_group
+Feature: Manage tax
+  As an employee
+  I must be able to add, edit and delete tax rules group
+
+  Scenario: Adding new tax rules group
+    When I add new tax rules group "sales-tax-rules-group" with following properties:
+      | name         | My Sales Tax Rules Group |
+      | is_enabled   | true                     |
+    Then tax rules group "sales-tax-rules-group" name should be "My Sales Tax Rules Group"
+    And tax rules group "sales-tax-rules-group" should be enabled
+
+  Scenario: Editing tax rules group
+    When I edit tax rules group "sales-tax-rules-group" with following properties:
+      | name         | The Sales Tax Rules Group |
+      | is_enabled   | false            |
```

Comment:
```suggestion
      | is_enabled   | false                     |
```
ctrl+alt+L should fix the table styles & indent (if using phpstorm)

Reply:
Fixed